marshal.go

```diff
@@ -180,7 +179,14 @@ func marshalOpenssh(pk interface{}, opts *MarshalOptions) ([]byte, error) {
 		blocksize = aes.BlockSize
 	}
 
-	check := mrand.Uint32()
+	// Get a crypto rand in the range [0, uint32-max)
```

Comment:
Comment doesn't align with code.  The range here is `[0, uint32max]`

Reply:
Actually the rand.Int does the non-inclusive upper bound thing

Reply:
right...but you're adding 1

Reply:
Never mind, let me fix it